eng/pipelines/templates/jobs/ci.yml

```diff
@@ -137,6 +137,10 @@ jobs:
               /p:ArtifactsPackagesDir=$(Build.ArtifactStagingDirectory) `
               /p:ServiceDirectory=* `
               /p:ProjectListOverrideFile=$(ProjectListOverrideFile) `
+              /p:IncludeSamples=false `
```

Comment:
Will it affect anything when my PR turning on IncludeSamples goes in?

Reply:
Yeah. It won't include them during `build`. This one is definitely one I was most expecting to pull back. I'll remove it.